pages/common/az-bicep.md

```diff
@@ -0,0 +1,33 @@
+# az bicep
+
+> Bicep CLI command group.
+> Part of `azure-cli`.
+> More information: <https://docs.microsoft.com/cli/azure/config>.
+
+- Build a Bicep file:
+
+`az bicep build --file {bicep_file}`
+
+- Attempt to decompile an ARM template file to a Bicep file:
+
+`az bicep decompile --file {json_template_file}`
```

Comment:
Why the description says `ARM template` but the example use `json_template_file`?
```suggestion
- Attempt to decompile an ARM template file to a Bicep file:

`az bicep decompile --file {{path/to/template_file.json}}`
```

Reply:
The ARM template is in json... I will change to:
```md
- Attempt to decompile an ARM template JSON file to a Bicep file:
```

Reply:
I see. Keep the original description is fine.

Reply:
Ok. Solved! Thanks!